zebedee-reader/src/main/java/com/github/onsdigital/zebedee/reader/api/endpoint/PublishedIndex.java

```diff
@@ -0,0 +1,44 @@
+package com.github.onsdigital.zebedee.reader.api.endpoint;
+
+import com.github.davidcarboni.restolino.framework.Api;
+import com.github.onsdigital.zebedee.reader.api.bean.PublishedIndexResponse;
+import com.github.onsdigital.zebedee.search.indexing.Document;
+import com.github.onsdigital.zebedee.search.indexing.FileScanner;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.ws.rs.GET;
+import java.io.IOException;
+import java.util.List;
+
+@Api
+public class PublishedIndex {
+
+    /**
+     * Retrieves list of content for endpoint <code>/publishedindex</code>
+     * <p>
+     * This endpoint returns a list of URIs for all the published content served by zebedee
+     *
+     * @param request  No authentication headers are required due to this only serving published content
+     * @param response Servlet response
+     * @return
+     * @throws IOException If an error occurs in processing data, typically to the filesystem, but also on the HTTP connection.
+     */
+    @GET
+    public PublishedIndexResponse read(HttpServletRequest request, HttpServletResponse response) throws IOException {
+
+        List<Document> docs = new FileScanner().scan();
```

Comment:
This method can throw an IOException - should we be handling that/logging the error?

Reply:
I feel it should be fine. IOException can be thrown due to failed http connection too and. Hence let it be handled by the calling client as with few other endpoints.